_rules/SC1-2-1-video-only-description-track.md

```diff
@@ -0,0 +1,107 @@
+---
+name: Video only element description track
```

Comment:
Shouldn't it be "video only element has description track"?

Reply:
Done. Thanks.